.github/workflows/lint.yml

```diff
@@ -1,8 +1,7 @@
 name: Lint
 
 on:
-  pull_request_target:
```

Comment:
This causes the lint to run against `main` rather than the branch `HEAD`. This means the previous behavior let unlinted code make it to `main`, but prevents CI from passing on the next PR.